devfile.yaml

```diff
@@ -0,0 +1,93 @@
+schemaVersion: 2.1.0
+metadata:
+  name: scala-sbt
+attributes:
+  che-theia.eclipse.org/sidecar-policy: USE_DEV_CONTAINER
+components:
+  - name: tools
+    container:
+      image: quay.io/devfile/universal-developer-image:ubi8-d433ed6
```

Comment:
```suggestion
      image: quay.io/devfile/universal-developer-image:ubi8-3df780e
```

Reply:
@sideeffffect It is needed to pick up your changes in developer image

Reply:
I changed to the latest one
https://quay.io/repository/devfile/universal-developer-image?tab=tags
thanks